src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx

```diff
@@ -215,6 +215,173 @@ In this example, if the incoming request is not associated with a valid user ses
 
 </Callout>
 
+### (Experimental) Perform authentication on the server side and enable HttpOnly cookies
+
+<Callout warning>
+
+**NOTE:** Once you enable the server-side sign-in feature, auth tokens are stored in HttpOnly cookies and you may not change the HttpOnly attribute. Since these cookies are inaccessible from client-side scripts, you won’t be able to use any Amplify JS APIs on the client side. Therefore, you don’t need to configure Amplify on the client side. You can keep using [these Amplify JS server-side APIs](/[platform]/build-a-backend/server-side-rendering/#supported-apis-for-nextjs-server-side-usage) on the server side.
```

Comment:
```suggestion
**Warning:** Once you enable the server-side sign-in feature, auth tokens are stored in HttpOnly cookies and you may not change the HttpOnly attribute. Since these cookies are inaccessible from client-side scripts, you won’t be able to use any Amplify JS APIs on the client side. Therefore, you don’t need to configure Amplify on the client side. You can keep using [these Amplify JS server-side APIs](/[platform]/build-a-backend/server-side-rendering/#supported-apis-for-nextjs-server-side-usage) on the server side.
```

nit for warning callout to use "Warning" text, align casing with other notes